src/ziggurat/config.clj

```diff
@@ -108,3 +110,61 @@
 (defn -get [^String key]
   (let [config-vals (get config (keyword key))]
     (java-response config-vals)))
+
+(def ^:private consumer-config-mapping-table
```

Comment:
for Consumer namespace, `:commit-interval-ms` should be mapped to `:auto-commit-interval-ms`

Reply:
naming conflict with streams